hack/lib/test.sh

```diff
@@ -35,13 +35,78 @@ kube::test::clear_all() {
   fi
 }
 
-# Prints the calling file and line number $1 levels deep
```

Comment:
All of that seems complicated, I was more thinking about creating a new function to match multiple strings 😄

Reply:
Heh, yeah, I guess I went down the rabbit hole a little on refactoring.  Let me revisit...

Reply:
OK, new push just adds `kube::test::get_object_assert_any()` and uses it.  Take a look now and let me know if this is more what you were thinking.